src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SmkStatementParsing.kt

```diff
@@ -63,19 +65,24 @@ class SmkStatementParsing(
             val actualToken = SnakemakeLexer.KEYWORDS[myBuilder.tokenText!!]
             if (actualToken != null) {
                 myBuilder.remapCurrentToken(actualToken)
+            } else if (isToplevelDecoratorKeyword()) {
+                myBuilder.remapCurrentToken(SmkTokenTypes.WORKFLOW_TOPLEVEL_DECORATOR_KEYWORD)
             }
         }
+
         val tt = myBuilder.tokenType
 
-        if (tt !in SmkTokenTypes.WORKFLOW_TOPLEVEL_DECORATORS) {
+        if (tt !in SmkTokenTypes.WORKFLOW_TOPLEVEL_DECORATORS &&
```

Comment:
Workflow top level decorators is supposed to include WORKFLOW_TOPLEVEL_DECORATOR_KEYWORD

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/405/commits/38d3b41b85c33d3f87cce080fa09d5b7a22db553